extensions/desktop/command-chain/run

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
isn't bash heavier?

Reply:
Yes, but you was already launching bash for hooks-configure-fonts and desktop-launch, so with this patch we are saving the two /bin/sh launched previously.

Reply:
Also, bash is required for those two scripts because they use some bash-specific syntax.